### DISCUSSION TITLE
Winking this allocator

### DISCUSSION THREAD

**Discussion by oreyg** [spaskalev/buddy_alloc#36] [General]
What would be an efficient strategy to clean this allocator, given the only things we want to retain are parameters we passed during initialization?

**Comment by spaskalev** [comment]
Without a code change just re-initialize it at the same location - this should zero out the allocation tree.

If you wish to release the allocations - you can use the `buddy_walk` function to iterate over them and call `buddy_free` for each one.

**Comment by oreyg** [comment]
Thank you :)
